frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -1821,13 +1821,13 @@ public function testAggregateFeedback() {
         $tags1 = array_map($extractName, $tagArrayForProblem1);
         $this->assertEquals(
             $tags1,
-            ['problemRestrictedTagLanguage', 'problemTopicDynamicProgramming', 'problemTopicGreedy', 'problemTopicMath', 'problemTopicMatrices']
+            ['problemLevelBasicIntroductionToProgramming','problemRestrictedTagLanguage', 'problemTopicDynamicProgramming', 'problemTopicGreedy', 'problemTopicMath', 'problemTopicMatrices']
```

Comment:
nit:

```suggestion
            [
                'problemLevelBasicIntroductionToProgramming',
                'problemRestrictedTagLanguage',
                'problemTopicDynamicProgramming',
                'problemTopicGreedy',
                'problemTopicMath',
                'problemTopicMatrices',
            ]
```

esta línea ya es muy grande :S igual abajo.